spec/queries/find_identifiers_to_reconcile_spec.rb

```diff
@@ -19,10 +20,15 @@
       change_set.validate(source_metadata_identifier: "123456")
       saved_resource = change_set_persister.save(change_set: change_set)
 
+      change_set = ScannedResourceChangeSet.new(resource3)
```

Comment:
Can you update the text portion (`it (...)`) to encompass the new requirement?

Reply:
it's not super clear from the test setup why resource3 should not be retrieved because the state is taken as the default value.